gravitee-node-certificates/src/main/java/io/gravitee/node/certificates/ReloadableKeyManager.java

```diff
@@ -39,13 +40,18 @@ public class ReloadableKeyManager extends X509ExtendedKeyManager {
     private static final Logger logger = LoggerFactory.getLogger(ReloadableKeyManager.class);
 
     static final int MAX_SNI_DOMAINS = 10000;
+    private final String serverId;
 
     private String defaultAlias;
 
     private Map<String, String> sniDomainAliases;
-    private volatile X509ExtendedKeyManager delegate;
+    private final AtomicReference<X509ExtendedKeyManager> delegate = new AtomicReference<>();
```

Comment:
I'm wondering if we need an atomic reference here. I'm not even convinced anymore that "volatile" was required here (same for the truststore part).
For performance consideration, I would use a regular attribute. WDYT?